### PR TITLE
Dependencies: bump `go-azure-sdk` to `v0.20220815.1092453`

### PR DESCRIPTION
**Pull request by jackofallops**

### CODE REVIEW

File: internal/services/policy/client/client.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ func NewClient(o *common.ClientOptions) *Client {
 	setDefinitionsClient := policy.NewSetDefinitionsClientWithBaseURI(o.ResourceManagerEndpoint, o.SubscriptionId)
 	o.ConfigureClient(&setDefinitionsClient.Client, o.ResourceManagerAuthorizer)
 
-	policyInsightsClient := policyinsights.NewPolicyInsightsClientWithBaseURI(o.ResourceManagerEndpoint)
+	policyInsightsClient := remediations.NewRemediationsClientWithBaseURI(o.ResourceManagerEndpoint)
```
Comment (stephybun) [inline internal/services/policy/client/client.go:33]: Minor but for consistency
```suggestion
	remediationsClient := remediations.NewRemediationsClientWithBaseURI(o.ResourceManagerEndpoint)
```

File: internal/services/policy/client/client.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ func NewClient(o *common.ClientOptions) *Client {
 		DefinitionsClient:                   &definitionsClient,
 		ExemptionsClient:                    &exemptionsClient,
 		SetDefinitionsClient:                &setDefinitionsClient,
-		PolicyInsightsClient:                &policyInsightsClient,
+		RemediationsClient:                  &policyInsightsClient,
```
Comment (stephybun) [inline internal/services/policy/client/client.go:44]: Same here
```suggestion
		RemediationsClient:                  &remediationsClient,
```

File: internal/services/videoanalyzer/client/client.go [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
+	EdgeModuleClient     *edgemodules.EdgeModulesClient
+	VideoAnalyzersClient *videoanalyzers.VideoAnalyzersClient
 }
 
 func NewClient(o *common.ClientOptions) *Client {
-	VideoAnalyzersClient := videoanalyzer.NewVideoAnalyzerClientWithBaseURI(o.ResourceManagerEndpoint)
+	edgeModulesClient := edgemodules.NewEdgeModulesClientWithBaseURI(o.ResourceManagerEndpoint)
+	o.ConfigureClient(&edgeModulesClient.Client, o.ResourceManagerAuthorizer)
 
+	VideoAnalyzersClient := videoanalyzers.NewVideoAnalyzersClientWithBaseURI(o.ResourceManagerEndpoint)
```
Comment (stephybun) [inline internal/services/videoanalyzer/client/client.go:18]:
```suggestion
	videoAnalyzersClient := videoanalyzers.NewVideoAnalyzersClientWithBaseURI(o.ResourceManagerEndpoint)
```

File: internal/services/videoanalyzer/client/client.go [hunk 4]
```diff
@@ -1,20 +1,25 @@
...
 	o.ConfigureClient(&VideoAnalyzersClient.Client, o.ResourceManagerAuthorizer)
 
 	return &Client{
+		EdgeModuleClient:     &edgeModulesClient,
 		VideoAnalyzersClient: &VideoAnalyzersClient,
```
Comment (stephybun) [inline internal/services/videoanalyzer/client/client.go:23]:
```suggestion
		EdgeModuleClient:     &edgeModulesClient,
		VideoAnalyzersClient: &videoAnalyzersClient,
```

**Review by stephybun — APPROVED** [review]
Barring any test failures LGTM 🦆

**Comment by jackofallops** [comment]
Tests across affected services passed as normal.

**Pull request merged**